azure-pipelines-cloud-db-integration.yml

```diff
@@ -33,9 +27,10 @@ stages:
 
           - script: |
               pip install -r requirements-dev.txt --constraint constraints-dev.txt
-
             displayName: 'Install dependencies'
-
+          - script: |
+              pip install google-cloud-bigquery-storage
```

Comment:
Should we pin this? Or are we okay with the latest? Just want to double check that this won't cause any incompatibility issues.

Reply:
I think I'm ok with using the latest for now, since we are using a pretty recent version of Python for these tests. Are you ok with that too?


Here is a note from [the package site](https://pypi.org/project/google-cloud-bigquery-storage/#description)

`The last version of this library compatible with Python 2.7 and 3.5 is google-cloud-bigquery-storage==1.1.0.`